test/jiraIssueStatusUpdates/issueTransitioner.test.js

```diff
@@ -9,132 +9,163 @@ beforeEach(() =>  {
     jiraApiMock.resetMocks();
 });
 
-test("Dealing with transitioning issues error cases", () => {
```

Comment:
With the refactor to add the "getTransitionStatus" method its needed the tests refactoring.